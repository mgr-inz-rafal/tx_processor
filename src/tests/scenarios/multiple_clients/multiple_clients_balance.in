type,client,tx,amount
deposit,1,1,1
deposit,2,2,2
deposit,1,3,3
withdrawal,1,4,0.1
withdrawal,2,5,0.2
deposit,2,6,4
withdrawal,1,7,0.3
withdrawal,2,8,0.4