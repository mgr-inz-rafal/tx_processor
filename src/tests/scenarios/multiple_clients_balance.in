type,client,tx,amount
deposit,1,1,1
deposit,2,2,2
deposit,1,3,3
withdrawal,1,5,0.1
withdrawal,2,6,0.2
deposit,2,7,4
withdrawal,1,8,0.3
withdrawal,2,9,0.4