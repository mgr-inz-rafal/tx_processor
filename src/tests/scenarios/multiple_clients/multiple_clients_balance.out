client,available,held,total,locked
1,3.6,0,3.6,false
2,5.4,0,5.4,false